docs/CLI.md

```diff
@@ -101,3 +101,11 @@ If you want to export extra mAP bar plots and annotation area stats add `--extra
 If you want to specify area regions, set it as `--areas 1024 9216 10000000000`.
 
 If you want to specify export directory, set it as `--out_dir output/folder/directory `.
+
+## Custom scripts
+
+All scripts can be downloaded from [scripts directory](sahi/scripts) and modified by your needs. After installing `sahi` by pip, all scripts can be call from any directory as:
+
+```bash
+python script_name.py
```

Comment:
I didn't get it 😅

Reply:
if you want to modify these scripts, you cannot do them with the cli, you have to download the scripts, update it and run as python script_name.py :sweat_smile: